friends/src/App.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './css/index.css';
+import axios from 'axios';
+import FriendsContainer from './components/FriendsContainer';
+
+class App extends Component {
+  constructor(){
+    super();
+    this.state={
+      friends:[]
+    }
+  }
+
+  componentDidMount() {
+    axios
+      .get('http://localhost:5000/friends')
+      .then(response => {
+        this.setState({ friends: response.data })
+      })
+      .catch(error => {});
```

Comment:
Would like to see some action taken if an error is sent back from the server. Preferably some way to tell the user that something went wrong.